README.md

```diff
@@ -590,14 +590,15 @@ var redis = new Redis({
 
 ## Sentinel
 ioredis supports Sentinel out of the box. It works transparently as all features that work when
-you connect to a single node also work when you connect to a sentinel group. Make sure to run Redis >= 2.8.12 if you want to use this feature.
+you connect to a single node also work when you connect to a sentinel group. Make sure to run Redis >= 2.8.12 if you want to use this feature. Sentinels have a default port of 26379.
 
 To connect using Sentinel, use:
 
 ```javascript
 var redis = new Redis({
   sentinels: [{ host: 'localhost', port: 26379 }, { host: 'localhost', port: 26380 }],
-  name: 'mymaster'
+  name: 'mymaster',
+  db: 0
```

Comment:
`db` option is not sentinel-specified and it's documented here: https://github.com/luin/ioredis/blob/master/API.md#new-redisport-host-options. What problem did you encounter?